api/alias_api/__init__.py

```diff
@@ -231,5 +231,21 @@ def import_alias_api():
 
 #
 # Import the Alias Python API module
-#
-import_alias_api()
+
+# For OpenModel, we need to ensure that the Alias DLLs are in the search path.
+# OpenAlias does not have to worry about this because all the DLLs are in the
+# exe path, which is automatically added to the search path.
+if os.path.basename(sys.executable) != "Alias.exe" and hasattr(os, "add_dll_directory"):
+
+    # get the path to the Alias exec bin folder. It should be first in the PATH environment variable
```

Comment:
This comment needs to be updated since the bin folder is not found from the path anymore